packages/cli/src/lib/snapshot/snapshot.ts

```diff
@@ -138,45 +139,45 @@ export class Snapshot {
     });
   }
 
-  public async waitUntilDone(
-    operationToWaitFor?: ResourceSnapshotsReportType | null,
-    options: WaitUntilDoneOptions = {},
-    iteratee = (_report: ResourceSnapshotsReportModel) => {}
+  public waitUntilDone(
+    operationToWaitFor: ResourceSnapshotsReportType | null,
+    options: Partial<WaitUntilDoneOptions> = {},
```

Comment:
Given the spread l145, we can provide a `Partial`. This allow the caller to only set the options he want